osu.Game/Screens/Play/Player.cs

```diff
@@ -161,9 +161,9 @@ private void addGameplayComponents(Container target, WorkingBeatmap working)
                 .WithChild(beatmapSkinProvider
                     .WithChild(target = rulesetSkinProvider)));
 
-            target.AddRange(new Drawable[]
+            target.AddRange(new[]
             {
-                DrawableRuleset,
+                DrawableRuleset as Drawable,
```

Comment:
this is the reason for the xmldoc that you removed. casting an interface to a drawable like this is super-dangerous.

Reply:
but it's an `IDrawable`, it can't be inherited from a normal class, no?

Reply:
there's nothing in place to enforce as such, which is why we weren't using an interface until now. if you can explain what this change is for (ie. what is dependent on switching to an interface) that may help figure a correct direction here.